jax/_src/ad_checkpoint.py

```diff
@@ -544,10 +545,13 @@ def remat_partial_eval(trace, *tracers, jaxpr, **params):
   jaxpr_known, in_used_known = pe.dce_jaxpr(jaxpr_known, out_used_known)
   num_res = sum(used_res)
 
+  # insert reduce_precision calls
```

Comment:
I'm going to ask for a slightly better comment, or at least a link to the excellent PR.